README.md

```diff
@@ -15,59 +15,57 @@ and using:
 * For examples, see the [examples directory](https://github.com/google/google-api-go-client/tree/master/examples).
 * For support, use the [golang-nuts](https://groups.google.com/group/golang-nuts) mailing list.
-* The code review instance may be found [here](https://code-review.googlesource.com).
 
 ## Status
+
 [![Go Reference](https://pkg.go.dev/badge/google.golang.org/api.svg)](https://pkg.go.dev/google.golang.org/api)
 
-These are auto-generated Go libraries from the Google Discovery Service's JSON description files of the available "new style" Google APIs.
+These are auto-generated Go libraries from the Google Discovery Service's JSON description files of the available "new
+style" Google APIs.
 
-Due to the auto-generated nature of this collection of libraries, complete APIs or specific versions can appear or go away without notice.
-As a result, you should always locally vendor any API(s) that your code relies upon.
+Due to the auto-generated nature of this collection of libraries, complete APIs or specific versions can appear or go
+away without notice. As a result, you should always locally vendor any API(s) that your code relies upon.
 
-These client libraries are officially supported by Google.  However, the libraries are considered complete and are in maintenance mode. This means that we will address critical bugs and security issues but will not add any new features.
+These client libraries are officially supported by Google.  However, the libraries are considered complete and are in
+maintenance mode. This means that we will address critical bugs and security issues but will not add any new features.
 
-If you're working with Google Cloud Platform APIs such as Datastore or Pub/Sub,
-consider using the
-[Cloud Client Libraries for Go](https://github.com/googleapis/google-cloud-go)
-instead. These are the new and
-idiomatic Go libraries targeted specifically at Google Cloud Platform Services.
+If you're working with Google Cloud Platform APIs such as Datastore or Pub/Sub, consider using the
+[Cloud Client Libraries for Go](https://github.com/googleapis/google-cloud-go) instead. These are the new and idiomatic
+Go libraries targeted specifically at Google Cloud Platform Services.
 
-The generator itself and the code it produces are beta. Some APIs are
-alpha/beta, and indicated as such in the import path (e.g.,
-"google.golang.org/api/someapi/v1alpha").
+The generator itself and the code it produces are beta. Some APIs are alpha/beta, and indicated as such in the import
+path (e.g., "google.golang.org/api/someapi/v1alpha").
 
 ## Application Default Credentials Example
 
-Application Default Credentials provide a simplified way to obtain credentials
-for authenticating with Google APIs.
+Application Default Credentials provide a simplified way to obtain credentials for authenticating with Google APIs.
```

Comment:
Without already knowing what ADC is (even if you do), this section is very confusing.

The important part is what a user needs to do to authenticate, right?

Reply:
RIght, and the hope really is they have to do nothing. I will try to update the examples to modernize them.